src/validation/error.messages.ts

```diff
@@ -37,6 +37,7 @@ export enum ErrorMessages {
   START_DATE_MUST_BE_AFTER_DOB = "Start date must be on or after date of birth",
   UNABLE_TO_RETRIEVE_ENTITY_NUMBER = "Unable to retrieve entity number",
   UNABLE_TO_RETRIEVE_EXPECTED_DATE = "Unable to retrieve the entity's expected date",
+  MISSING_NATURE_OF_CONTROL = "Natures of control options may have changed since you the last time you saved your update statement. Select Change for each beneficial owner to check their natures of control are correct:",
```

Comment:
This message doesn't match the text in AC5, which will need updating. Is it agreed that we will now show a list of BOs?

Reply:
The message is in the comment

Reply:
Message in the comment doesn't have the colon (:) and I think we need to make clear in the AC what the error message will look like when we have multiple BOs of different types that have missing NOCs - i.e. what's displayed, first name, last name, company name.

Reply:
AC5 updated